src/azul/openapi/schema.py

```diff
@@ -220,10 +220,19 @@ def with_default(default: PrimitiveJSON, /, type_: Optional[TYPE] = None) -> JSO
         "format": "double",
         "default": 0
     }
+    >>> assert_json(with_default(10, min=10,max=20))
```

Comment:
```suggestion
    >>> assert_json(with_default(10, min=10, max=20))
```

Reply:
min max vs minimum maximum?

Reply:
I assumed these are just doc tests, and is to just showcase how it works? I'll change them.

Reply:
In that case I would have used something obviously meaningless like "foo" and "bar". Min and max were in the twilight zone between real and fantasy, which could be confusing. But this is fine.